WalletWasabi.Gui/Tabs/WalletManager/RecoverWalletViewModel.cs

```diff
@@ -101,6 +101,11 @@ public string Password
 			set => this.RaiseAndSetIfChanged(ref _password, value);
 		}
 
+		private void ValidatePasswordN(ErrorDescriptor obj)
```

Comment:
This method is not used anywhere should it be removed?.
Why its name is `ValidatePasswordN` with `N`?

Reply:
Forgot to rename that back sorry :)